configure.ac

```diff
@@ -85,9 +85,9 @@ dnl      age.
 dnl   6. If any interfaces have been removed since the last public release, then set age
 dnl      to 0.
 
-CLEAN_VERSION=`echo $PACKAGE_VERSION | $SED "s/p.*//"`
-VERSION_MAJOR=`echo $PACKAGE_VERSION | $SED "s/\..*//"`
-VERSION_MINOR=`echo $CLEAN_VERSION | $SED "s/.*\.//"`
+CLEAN_VERSION="1.0.30"
+VERSION_MAJOR="1"
+VERSION_MINOR="30"
 GEN_TOOL="autoconf"
 
 SHARED_VERSION_INFO="$VERSION_MAJOR:$VERSION_MINOR:0"
```

Comment:
as an aside, libtool's versioning doesn't actually work like this, it's more subtle: https://flameeyes.blog/2009/04/22/shared-object-version/
https://autotools.io/libtool/version.html